django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
On Tue, Jan 5, 2016 at 8:52 PM Carl Meyer <notifications@github.com> wrote:

> Making the hasher-selection smart enough to discard unusable hashers from
> the top of the list seems like a reasonable improvement


Security wise I do not like that idea. For example: Argon2 is the first in
the list for one of your deployments, you setup a new server (and switch to
it) which accidentally misses argon2 libs and your passwords get
downgraded… Chances that you will notice that anytime soon are very slim
imo.